content/en/about/deployment/index.md

```diff
@@ -14,17 +14,94 @@ doc_type: about
 
 {{< centered_block >}}
 
-## Step one
+You have decided you want to use Istio. Welcome to the world of service mesh. Congratulations, you're in great company!
 
-Istio addresses the challenges developers and operators face as monolithic applications transition towards a distributed microservice architecture. To see how, it helps to take a more detailed look at Istio’s service mesh.
+If you haven't already, you might like to try Istio out in a test environment, and run through our [Getting Started guide](../../docs/setup/getting-started/). This will give you an idea for the **traffic management**, **security** and **observability** features.
 
-## Step two
+## Do it yourself, or bring a guide?
 
-While the easy transition to mTLS for all microservices was a strong incentive, Istio also has the backing of many large organizations. Auto Trader UK was already working with Google, so knowing that Google was a strong backer of Istio gave them confidence it would be supported and grow long term.
+Istio is open-source software which you can download and install yourself. Getting a mesh installed on a Kubernetes cluster is as simple as running one command:
 
-Early success with experiments on GKE with Istio led to quick buy-in from the business. Capabilities that they had been trying to implement for months were suddenly ready in just a week. Istio was able to not only provide mTLS, but also robust retry and backup policies and outlier detection.
+{{< text bash >}}
+$ istioctl install
+{{< /text >}}
 
-## Step three
+As new versions are released, you can test them and gradually roll them out across your clusters.
+
+Many managed Kubernetes service providers have an option to automatically install and manage Istio for you. Check out our [distributors page](../about/ecosystem/) to see if your vendor supports Istio.
```

Comment:
```suggestion
Many managed Kubernetes service providers have an option to automatically install and manage Istio for you. Check out our [distributors page](/about/ecosystem/) to see if your vendor supports Istio.
```

Reply:
That'll fix the lint.

Reply:
Okay, I'll commit that and retry. This is sort of confusing because when I was running the lint locally, it kept telling me that I had to use relative links and not absolute links.